src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -1,15 +1,19 @@
 package mate.academy.service;
 
-import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LogManager.getLogger();
+
     @Override
-    public User login(String login, String password) throws AuthenticationException {
+    public User login(String login, String password) {
         //TODO: add corresponding log message about method login was called
+        logger.info("Login method was called with user: {}", login);
         User user = findByLogin(login);
         if (!user.getPassword().equals(password)) {
-            throw new AuthenticationException("Username or password are incorrect");
+            logger.error("Failed login attempt for user '{}': incorrect password.", login);
         }
```

Comment:
The current implementation logs an error for incorrect passwords but does not throw an `AuthenticationException`. According to the task requirements, you should throw an `AuthenticationException` when the password is incorrect.